eng/scripts/meta_generation.ps1

```diff
@@ -0,0 +1,27 @@
+function ShouldGenerate-AutorestConfig {
+    Param(
+        [string] $autorestPath
+    )
+    $metaPath = $autorestPath.Replace("autorest.md", "_meta.json")
+    return !(Test-Path $autorestPath -PathType Leaf) && (Test-Path $metaPath -PathType Leaf)
```

Comment:
It looks like you are using the `&&` to return a single bool value here, as opposed to gating the second command on the success of the first. In that case, `&&` is another confusing powershell-ism. I think you might want `-and`:

```
$ $false && $true
False
True
$ $false -and $true
False
```